lib/ffmpeg/encoderbase.py

```diff
@@ -86,6 +86,8 @@ def gen_output_opts(self):
       opts += " -look_ahead_depth {ladepth}"
     if vars(self).get("vforced_idr", None) is not None:
       opts += " -force_key_frames expr:1 -forced_idr 1"
+    if vars(self).get("vintr_ref", None) is not None:
+      opts += " -int_ref_type 1 -int_ref_cycle_size 4 -int_ref_cycle_dist 20"
```

Comment:
Why hardcoded values?  Maybe better to let test config set these parameter values.

Reply:
> Why hardcoded values? Maybe better to let test config set these parameter values.

How about doing like this time?
Moreover, hevc "-low_power 0" doesn't support Intra refresh, so I only add intra refresh when hevc "-low_power 1".

Reply:
ok, good.